how-to/use-theming-basic/client/src/apps.ts

```diff
@@ -0,0 +1,83 @@
+import type OpenFin from "@openfin/core";
+import { type App } from "@openfin/workspace";
+import { AppManifestType, getCurrentSync } from "@openfin/workspace-platform";
+
+/**
+ * Get the list of apps to display.
+ * @returns List of app definitions.
+ */
+export function getApps(): App[] {
+	return [OPENFIN_INFORMATION_APP];
+}
+
+/**
+ * App definition to use for demonstration which show OpenFin environment information.
+ */
+export const OPENFIN_INFORMATION_APP: App = {
+	appId: "openfin-information",
+	title: "OpenFin Information",
+	description: "Display information about the OpenFin environment",
+	manifest: "http://localhost:8080/common/views/platform/of-info.view.fin.json",
+	manifestType: "view",
+	icons: [
+		{
+			src: "http://localhost:8080/common/images/icon-blue.png"
+		}
+	],
+	contactEmail: "contact@example.com",
+	supportEmail: "support@example.com",
+	publisher: "OpenFin",
+	intents: [],
+	images: [
+		{
+			src: "http://localhost:8080/common/images/previews/of-info.png"
+		}
+	],
+	tags: ["view", "openfin"]
```

Comment:
can we add versions as a tag. Then it will fall under the openfin version related views. I am adding this to the common entry so it can be filtered out.

Reply:
Done